config/sidekiq.yml

```diff
@@ -56,9 +56,6 @@
     cron: "0 4 * * *"
     description: "For Publishers who have stale uphold codes or access parameters, clean out uphold codes or access parameter."
     queue: scheduler
-  TransferChannelsJob:
```

Comment:
this was just a duplicate entry - right?

Reply:
Yep!